src/flownet/ahm/_ahm_iteration_analytics.py

```diff
@@ -296,15 +294,18 @@ def make_dataframe_simulation_data(path, eclbase_file, keys):
         realizations_dict[runpath] = _load_simulations(runpath, eclbase)
 
     # Prepare dataframe
-    df = pd.DataFrame()
+    df_sim = pd.DataFrame()
     for id_real, runpath in enumerate(realizations_dict.keys()):
         df_tmp = pd.DataFrame()
-        dates = realizations_dict[runpath].dates
         if id_real == 0:
-            df["DATE"] = pd.Series(dates)
-            df["REAL_ID"] = pd.Series(id_real * np.ones(len(dates)), dtype=int)
-        df_tmp["DATE"] = pd.Series(dates)
-        df_tmp["REAL_ID"] = pd.Series(id_real * np.ones(len(dates)), dtype=int)
+            df_sim["DATE"] = pd.Series(realizations_dict[runpath].dates)
+            df_sim["REAL_ID"] = pd.Series(
+                id_real * np.ones(len(realizations_dict[runpath].dates)), dtype=int
+            )
+        df_tmp["DATE"] = pd.Series(realizations_dict[runpath].dates)
+        df_tmp["REAL_ID"] = pd.Series(
+            id_real * np.ones(len(realizations_dict[runpath].dates)), dtype=int
+        )
```

Comment:
What is the reasoning behind not creating the variable `dates` (which is used some times in this code block)?

Reply:
`pylint` was complaining of "too many locals"

Reply:
Ah, ok. Maybe we can keep it, and simply (at least for now) disable that pylint message in this function? 🙂 
http://pylint.pycqa.org/en/latest/faq.html#is-it-possible-to-locally-disable-a-particular-message